### ISSUE TITLE
Cannot import babel helper

### ISSUE THREAD

**Issue by ptondereau** [One-com/react-truncate#17]
Hello!

When I try to use your component on my React project, it throws an error:

```
Uncaught Error: Cannot find module 'babel-runtime/helpers/objectWithoutProperties' from 'react-truncate/lib/Truncate.js'
```

`package.json`:

```
"dependencies": {
    "autosize": "^3.0.15",
    "babel-brunch": "^6.0.6",
    "babel-core": "^6.14.0",
    "babel-loader": "^6.2.5",
    "babel-plugin-add-module-exports": "^0.2.1",
    "babel-plugin-react-display-name": "^2.0.0",
    "babel-plugin-transform-class-properties": "^6.11.5",
    "babel-plugin-transform-runtime": "^6.12.0",
    "babel-polyfill": "^6.13.0",
    "babel-preset-es2015": "^6.14.0",
    "babel-preset-es2016": "^6.11.3",
    "babel-preset-es2017": "^6.14.0",
    "babel-preset-react": "^6.5.0",
    "babel-preset-stage-0": "^6.5.0",
    "babel-runtime": "^6.11.6",
    "babel-template": "^6.14.0",
    "bower": "^1.7.7",
    "brunch": "https://github.com/spyl94/brunch/tarball/6f20a214104f8db7842365b6063c18ebc65134f9",
    "classnames": "^2.2.3",
    "clean-css-brunch": "^2.0.0",
    "css-brunch": "^2.0.0",
    "diff": "^3.0.0",
    "exports-loader": "^0.6.3",
    "fancybox": "^3.0.0",
    "flat": "^2.0.1",
    "flux": "^2.1.1",
    "imports-loader": "^0.6.5",
    "javascript-brunch": "^2.0.0",
    "jquery": "^3.1.0",
    "json-brunch": "^1.5.4",
    "mailcheck": "^1.1.1",
    "moment": "^2.11.2",
    "process-env-brunch": "^1.4.4-d",
    "quill": "^0.20.1",
    "react": "^15.3.1",
    "react-bootstrap": "^0.29.5",
    "react-checkbox-group": "https://github.com/spyl94/react-checkbox-group/tarball/296721a62a682c5e441bdd449704550b69257af9",
    "react-dnd": "^2.1.4",
    "react-dnd-html5-backend": "^2.1.2",
    "react-dom": "^15.3.1",
    "react-dropzone": "^3.3.4",
    "react-google-charts": "^1.0.2",
    "react-google-recaptcha": "^0.5.2",
    "react-intl": "1.2.0",
    "react-linkify": "^0.1.1",
    "react-nl2br": "^0.2.0",
    "react-on-rails": "^6.1.0",
    "react-radio": "^2.1.1",
    "react-redux": "^4.4.5",
    "react-router": "^2.7.0",
    "react-router-bootstrap": "^0.23.1",
    "react-truncate": "^1.1.4",
    "readmore": "0.0.2",
    "redux": "^3.4.0",
    "redux-form": "6.0.0-rc.3",
    "redux-saga": "^0.11.0",
    "sass-brunch": "2.6.2",
    "sinon": "^1.17.3",
    "uglify-js-brunch": "^2.0.1",
    "webpack": "^1.13.2",
    "whatwg-fetch": "^1.0.0"
  },
```

`.babelrc`:

```
{
  "presets": ["react", "es2015", "es2016", "es2017", "stage-0"],
  "plugins": [
    "add-module-exports"
  ]
}
```

**Comment by PabloSichert** [comment]
Did you by chance switch around node or npm versions - could you try `rm -rf node-modules && npm install`?

**Comment by ptondereau** [comment]
Yes i checked...

I try to reinstall several time but no change  :/

**Comment by PabloSichert** [comment]
I found the error - the `transform-runtime` babel plugin put some dependencies into the bundled code that were only needed for development.

Fixed as of `v1.1.5`.

Thank you for reporting this! :sun_with_face:

**Comment by ptondereau** [comment]
sweet 👍many thanks
